src/GridFS/Bucket.php

```diff
@@ -17,6 +17,7 @@
 
 namespace MongoDB\GridFS;
 
+use MongoDB\Collection;
```

Comment:
It looks like this was needed for the `getChunksCollection` return type. Good catch.